getambassadorio_blc.py

```diff
@@ -277,10 +279,19 @@ def pump():
 
 if __name__ == "__main__":
     try:
-        if len(sys.argv) < 2:
-            print(f"Usage: {sys.argv[0]} PROJDIR", file=sys.stderr)
+        if len(sys.argv) < 3:
+            print(
+                f"Usage: {sys.argv[0]} PROJDIR PAGES_TO_CHECK BASE_ADDRESS", file=sys.stderr
+            )
             sys.exit(2)
-        sys.exit(main(AmbassadorChecker, sys.argv[1], sys.argv[2] if len(sys.argv) else ''))
+        sys.exit(
+            main(
+                AmbassadorChecker,
+                sys.argv[1],
+                sys.argv[2],
+                sys.argv[3] if len(sys.argv) else '',
```

Comment:
Can you explain what this does.  I think that before it would pass '' as the third argument if `len(sys.argv)` is not 0, but that has changed.

However, I don't understand why would you want to do that.  Maybe there is a bug in that if?

Reply:
I can probably just remove that check. I don't see instances where we pass fewer args to this script

Reply:
Really it's just called from the Makefile

Reply:
I think that makes sense